hack/build/build-cdi-func-test-registry-host.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+#Copyright 2018 The CDI Authors.
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+
+set -euo pipefail
+
+script_dir="$(readlink -f $(dirname $0))"
+source "${script_dir}"/common.sh
+source "${script_dir}"/config.sh
+
+TEST_REGISTRY_HOST_POPULATE="cdi-func-test-registry-populate"
+TEST_REGISTRY_HOST="cdi-func-test-registry"
+TEST_REGISTRY_INIT="cdi-func-test-registry-init"
```

Comment:
Aren't we defining FUNC_TEST_REGISTRY and FUNC_TEST_REGISTRY_INIT in config.sh with the same values? Can't we just use those instead of defining the same value again with a different name?

Reply:
Done